### DISCUSSION TITLE
WSL2 Github Copilot Chat - no icon vscode.editorChat.start error

### DISCUSSION THREAD

**Discussion by mcmiv413** [community/community#57943] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am running WSL2 and using vscode-insiders, github copilot is installed and working fine but I can't get the Chat to load. 

`Menu item references a command vscode.editorChat.start which is not defined in the commands section.`

On the release version I get this:

`Menu item references a command interactiveEditor.start which is not defined in the commands section.`

I have tried to remove and re-install, I have also switched between the release and pre-release version as well as tried to manually downgrade the release version. I'd appreciate some advice on what to do next.

**Comment by conrizzo** [comment]
Here is a screenshot:


![image](https://github.com/community/community/assets/13004925/f26f4c6c-0b17-498f-a9a3-e974a73a5c32)


Seems I have a similar issue as well. Am also running vscode-insiders. I wonder if this is isolated to some, or if it is a larger issue?

**Comment by conrizzo** [comment]
Also maybe something to give a try, pressing CTRL+ALT+I (uppercase i) is a shortcut to open the chat window. This seems to get it working for me even with the message error.

**Reply by nbweir** [reply]
This worked for me, however that was not the default shortcut on my computer. After checking the keyboard shortcuts (Preferences: Open Keyboard Shortcuts) and updating "View: Show Chat" ("Workbench.panel.chatSidebar.copilot"), it worked.

**Comment by adrianforte** [comment]
I have the same issue as well. In my case, I tried opening the chat windows using the shortcut but nothing happens.

**Reply by dchenhe** [reply]
me too nothing happens....
